iothub/service/tests/RetryPolicies/IotHubServiceFixedDelayRetryPolicyTests.cs

```diff
@@ -33,5 +33,33 @@ public void FixedDelayRetryPolicy_IsFixedDelay(uint retryCount)
             // assert
             retryInterval.Should().Be(expected);
         }
+
+        [TestMethod]
+        public void FixedDelayRetryPolicy_UseJitter()
+        {
+            // arrange
+            uint retryCount = 10;
+            var expected = TimeSpan.FromSeconds(10);
+            var retryPolicy = new IotHubServiceFixedDelayRetryPolicy(0, expected, true);
+
+            // act
+            retryPolicy.ShouldRetry(retryCount, new IotHubServiceException("") { IsTransient = true }, out TimeSpan retryInterval);
+            retryInterval.Should().BeCloseTo(expected, TimeSpan.FromMilliseconds(500));
+        }
+
+        [TestMethod]
+        public void FixedDelayRetryPolicy_ShouldRetry_IsFalse()
+        {
+            // arrange
+            uint retryCount = 10;
+            var expected = TimeSpan.FromSeconds(10);
+            var retryPolicy = new IotHubServiceFixedDelayRetryPolicy(0, expected);
+
+            // act
+            bool shouldRetry = retryPolicy.ShouldRetry(retryCount, new Exception(), out TimeSpan retryDelay);
```

Comment:
nit: `new IotHubServiceException()` would also be non-retryable, right? Could we use that instead of `new Exception()` and add a comment explaning why shouldRetry() is false?